### DISCUSSION TITLE
Displaying text along with RDK field

### DISCUSSION THREAD

**Discussion by wsr017** [jspsych/jsPsych#763] [Q&A] [answered]
Hello everyone,

I am wondering if there is a way to include text above a dot motion field created by the jsPsych-rdk plugin. I am trying to show a sample dot motion field with text above it that says something along the lines of "This is an example of leftward motion" to describe the coherent direction. 

Is this possible by manipulating the rdk plugin, or is it only able to display the dot motion field? 

Thanks so much for the help!

**Comment by vijaymarupudi** [comment, marked as answer]
https://github.com/jspsych/jsPsych/blob/83980085ef604c815f0d97ab55c816219e969b84/plugins/jspsych-rdk.js#L355

This is where you would need to modify the plugin. Before that part add

```javascript
var text = document.createElement('div')
text.innerHTML = trial.text
display_element.appendChild(text)
```

Then you can pass in a text parameter for the plugin and it should display above the canvas I think.

**Comment by wsr017** [comment]
This worked perfectly! Thank you so much for the quick response!
